react/features/base/devices/functions.js

```diff
@@ -67,6 +67,34 @@ export function getDeviceIdByLabel(state: Object, label: string, kind: string) {
     }
 }
 
+/**
+ * Finds a device with a label that matches the passed id and returns its label.
```

Comment:
Side musing: this makes me wonder if we even need to store deviceId at all and instead can rely solely on label. Based on what you've seen would it be better to move to label over device id? There would likely need to be a translation layer, maybe in lib-jitsi-meet to convert label to id for the sake of calling gUM with an exact device id.

Reply:
Not a change for now, if we decide to do it ...

Reply:
That wasn't the question though.

Reply:
?